docs/new/references/flashblocks.md

```diff
@@ -0,0 +1,148 @@
+---
+description: Flashblocks support (alpha feature)
+---
+
+New support for "Flashblocks" is now available for alpha testing on Base Mainnet.
+
+ref: https://docs.base.org/base-chain/flashblocks/apps
+
+## Disclaimers
+
+* The only endpoint supporting Flashblocks is `https://base-mainnet-flash.streamingfast.io:443`.
+* That endpoint is not guaranteed to be stable or available at all times.
+* The protocol might still change in the next few weeks, as we gather feedback on usage.
+* Flashblocks coming from Firehose or Substreams endpoint should not be considered final data. Only write to database data coming from full blocks: there is no "undo" mechanism for Flashblocks.
+
+## Description
+
+* Flashblocks are partial blocks that are not fully confirmed yet. 
+* They are emitted every 200ms and contain a fraction of the transactions that will be in the final block.
+
+### Flashblocks in Substreams
+
+#### Partial Blocks
+
+* In Substreams, Flashblocks are called **partial blocks**, as a generalization of the concept, even though Flashblocks are the only supported implementation yet.
+* To benefit from partial blocks: 
+  1. you need the latest version of substreams CLI or library.
+  2. your substreams modules should avoid doing "block-level aggregations" and should only work on what is inside the "transactionTraces"
+
+Here's how it works:
+
+1. The "sequencer" emits a flashblock every 200ms (so a maximum of 10 per block height)
+2. The instrumented Base node reader sends the increasing versions of the same block to the Substreams engine and eventually, the full block.
+3. To keep up with the chain, it may skip a few emissions of partial blocks, but will never send the transactions out-of-order.
+4. The Substreams engine will remember what was processed for each active Substreams and only process the new transactions since the last execution.
+5. It sends the PartialData for each part of the full block as it gets it from the flash blocks
+6. If there is a reorg, new and undo signals are sent for the full blocks, but there is no consideration for the sent partial blocks. The user must always consider that the partial blocks data may become invalid.
+7. For this reason, there is no "cursor" sent with the partial blocks data.
+
+#### Changes to Protobuf models
+
+* Partial blocks are not sent as `BlockScopedData`, but as `PartialBlockData`, which is new possible type for `Response.message`
+
+```proto
+message PartialBlockData {
+  MapModuleOutput output = 1;
+  sf.substreams.v1.Clock clock = 2;
+  uint32 partial_index = 3;
+}
+```
+
+* The `sf.substreams.rpc.v2.Blocks/Request`  and `sf.substreams.rpc.v3.Blocks/Request` now contain these parameters:
+
+``` proto
```

Comment:
Need indents on your code block, when rendered, it's not aligned with the bullet

<img width="829" height="256" alt="Image" src="https://github.com/user-attachments/assets/b5567d13-cc78-4335-ba71-9b99a9ed1630" />